scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -31,18 +34,65 @@
                  "compute cluster, unable to get node from node name" node-name)
       false)))
 
+(defn num-waiting-synthetic-pods
```

Comment:
Nitpick: rename to `compute-num-waiting-synthetic-pods`

Reply:
Will do

Reply:
Done